### PR TITLE
gnoi 5.3 commit to experimental

### PR DESCRIPTION
**Pull request by nraghavendran**
test case to address https://github.com/openconfig/featureprofiles/issues/99. 
the test case needs to be made to work later.

### CODE REVIEW

File: feature/experimental/gnoi/copying_debug_files_test/copying_debug_files_test.go [hunk 1]
```diff
@@ -0,0 +1,97 @@
...
+	}
+	start := time.Now()
+	processKillResponse, err := gnoiClient.System().KillProcess(context.Background(), killProcessRequest)
+	if err != nil {
+		t.Fatalf("Failed to restart process %v with unexpected err: %v", process, err)
+	}
+
+	// Sleep for 60 seconds after process kill
+	t.Logf("gnoiClient.System().KillProcess() response: %v, err: %v", processKillResponse, err)
+	t.Logf("Time elapsed after process restart: ", time.Since(start))
```
Comment (sezhang2) [inline feature/experimental/gnoi/copying_debug_files_test/copying_debug_files_test.go:74]: Add %v

Comment (nraghavendran) [reply feature/experimental/gnoi/copying_debug_files_test/copying_debug_files_test.go]: resolved the comment

**Comment by sezhang2** [comment]
statistics failure:
=============

@@ -79,7 +79,7 @@
 	req := &hpb.GetRequest{
 		Path: &tpb.Path{
 			Elem: []*tpb.PathElem{
-				&tpb.PathElem{
+				{
 					Name: "components/component/state/software-version",
 				},
 			},

**Comment by nraghavendran** [comment]
changed to reflect other examples like net/model/operations/gnoice/healthz_test.go

**Comment by nraghavendran** [comment]
> # statistics failure:
> @@ -79,7 +79,7 @@ req := &hpb.GetRequest{ Path: &tpb.Path{ Elem: []*tpb.PathElem{
> 
> * ```
>     		&tpb.PathElem{
>   ```
> 
> * ```
>     		{
>     			Name: "components/component/state/software-version",
>     		},
>     	},
>   ```

I have changed this more, to reflect existing code for example in net/model/operations/gnoice/healthz_test.go

**Review by sezhang2 — APPROVED** [review]
Please commit the code after checks passed.

File: feature/experimental/gnoi/copying_debug_files_test/copying_debug_files_test.go [hunk 2]
```diff
@@ -0,0 +1,106 @@
...
+	}
+
+	// Sleep for 60 seconds after process kill
+	t.Logf("gnoiClient.System().KillProcess() response: %v, err: %v", processKillResponse, err)
+	t.Logf("Time elapsed after process restart: %v", time.Since(start))
+	t.Logf("Wait 60 seconds for process to restart ...")
+	time.Sleep(60 * time.Second)
+
+	// construct struct with values for GetRequest for healthz call
+	req = &hpb.GetRequest{
```
Comment (sezhang2) [inline feature/experimental/gnoi/copying_debug_files_test/copying_debug_files_test.go:79]: Error: vet: feature/experimental/gnoi/copying_debug_files_test/copying_debug_files_test.go:79:2: undeclared name: req
Error: Process completed with exit code 2.

**Review by sezhang2 — CHANGES_REQUESTED** [review]
Error: vet: feature/experimental/gnoi/copying_debug_files_test/copying_debug_files_test.go:79:2: undeclared name: req
Error: Process completed with exit code 2.

**Review by sezhang2 — APPROVED** [review]

**Pull request merged**